lib/Doctrine/ORM/Tools/SchemaValidator.php

```diff
@@ -134,9 +134,9 @@ public function validateClass(ClassMetadataInfo $class)
                             'field ' . $assoc['targetEntity'] . '#' . $assoc['inversedBy'] . ' which does not exist.';
                 } elseif ($targetMetadata->associationMappings[$assoc['inversedBy']]['mappedBy'] === null) {
                     $ce[] = 'The field ' . $class->name . '#' . $fieldName . ' is on the owning side of a ' .
-                            'bi-directional relationship, but the specified mappedBy association on the target-entity ' .
-                            $assoc['targetEntity'] . '#' . $assoc['mappedBy'] . ' does not contain the required ' .
-                            "'inversedBy' attribute.";
+                            'bi-directional relationship, but the specified inversedBy association on the target-entity ' .
+                            $assoc['targetEntity'] . '#' . $assoc['inversedBy'] . ' does not contain the required ' .
+                            "'mappedBy' attribute.";
```

Comment:
Another way to confirm that you are correct is to look at the block on line 114, which should be a mirror of this one.

Reply:
That's why I was thinking. I think tripped over that message long time ago and now I did it again, but this time I decided to fix it :)